### DISCUSSION TITLE
ALAppExtensions - Modules - DotNet Aliases : Assembly not found

### DISCUSSION THREAD

**Discussion by lorenzosoncini** [microsoft/ALAppExtensions#18469] [Q&A]
Good evening to everyone,
I am trying to build to build the module "ALAppExtensions / System / DotNet Aliases/" but I receive 59 error like this:
error AL0451: An assembly named 'Microsoft.Dynamics.Nav.MX, PublicKeyToken=null' could not be found in the assembly probing paths ...
I build on a OnPrem Business Central 19. In my settings JSON I have added the folder (BC Service Add-Ins):
{
    "al.assemblyProbingPaths": [
        "./.netpackages",
        "C:\\Windows\\assembly\"",
        "C:\\Program Files\\Microsoft Dynamics 365 Business Central\\190\\Service\\Add-ins\""
    ]

Where I can found all the DLL assembly? 

Thanks
Lorenzo Soncini
Italy

**Comment by JesperSchulz** [comment]
That looks about right!? It's also in line with the blog-posts there are to be found on the topic:
https://demiliani.com/2018/12/17/dynamics-365-business-central-on-premise-and-net-assemblies-things-to-know/
https://www.hougaard.com/dotnet-with-extensions-v2/
Not sure what's wrong here! Anyone else got any idea?
Unfortunately I don't have "normally installed" versions of BC - I'm using a special development setup, which is why I cannot really help here.
